install-dev/data/db_structure.sql

```diff
@@ -1919,50 +1919,6 @@ CREATE TABLE `PREFIX_range_weight` (
   )
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8mb4 COLLATION;
 
-/* Referrer stats */
-CREATE TABLE `PREFIX_referrer` (
```

Comment:
You need to update the SQL scripts in the autoupgrade module right? It's better to do it now or I'm sure we'll forget about it ^^

Reply:
Thanks. It's done : https://github.com/PrestaShop/autoupgrade#447